lib/templates.rake

```diff
@@ -103,6 +107,23 @@ def update_snippet
   { :diff => diff, :status => status, :result => result }
 end
 
+def get_default_branch
+  # Always use develop on Foreman-nightly
+  return "-b develop" if Foreman::Version.new.tag == "develop"
+
+  url = URI.parse("https://api.github.com/repos/theforeman/community-templates/branches")
```

Comment:
When you say "allow users with internal git repos to use multiple branches", how do you mean? Users can already specify a custom branch anyway, this is just to handle defaults for the templates repo transition